scripts/kudu

```diff
@@ -16,7 +16,10 @@ exec_name = "kudu"
 def get_system_suffix(system: str) -> Optional[str]:
     system = system.lower()
     if system == "linux":
-        return "bin/linux/"
+        if platform.processor() == "aarch64":
+            return "bin/linux/arm"
```

Comment:
A lot of the cloud providers have nicely priced arm instances. Are you worried about the distribution getting too large?

Reply:
In general, the good practice on such matters AFAIU is it host binaries on a server and download them on installation. This way, the repo is not blown up. Also, there is a hard limit on the file size, if we want to statically link kudu (which might make sense, it erases libc problems, for example), it can no longer be committed to the repo

Reply:
Good points,  I've removed it.

Reply:
Wait, it seems that you've deleted the wrong branch. Also, do we even need that PR now?

Reply:
I've fixed it, we do still need it because this version of kudu uses an older version of GLIBC, so it will work on older Linux distros.

Reply:
Okay, then let us just update the kudu binary then?